classes/ActionScheduler_QueueCleaner.php

```diff
@@ -26,7 +26,11 @@ public function delete_old_actions() {
 		) );
 
 		foreach ( $actions_to_delete as $action_id ) {
-			$this->store->delete_action( $action_id );
+			try {
+				$this->store->delete_action( $action_id );
+			} catch ( Exception $e ) {
+				do_action( 'action_scheduler_failed_old_action_deletion', $action_id, $e, $lifespan, count( $actions_to_delete ) );
```

Comment:
One minor thing here: I'd like to see a docblock for this action so that we can start having some inline documentation for the actions and filters.

Reply:
@JPry good idea. I see that's also part of WordPress' [PHP Documentation Standards](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters) now. I've fixed that up and rebased in the changed. 👍